go.mod

```diff
@@ -0,0 +1,56 @@
+module github.com/kubernetes-incubator/kube-aws
```

Comment:
We should be careful committing this until we shift the project to `GOMODULES`.

Reply:
Good point I'll remove both!